app/routes/task_routes.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, make_response, jsonify, request, abort
+from app.models.task import Task
+from app import db
+from datetime import datetime
+import requests
+from .route_helpers import validate_model
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@bp.route("", methods=["GET"])
+def get_all_tasks():
+    sorted_query = request.args.get("sort")
+    if sorted_query == "asc":
+        tasks = Task.query.order_by("title")
```

Comment:
Assuming this `order_by` does ascending by default, without you having to explicitly call `order_by(Task.title.asc())`? Also, can the `.all()` be left off of the `order_by` queries (i.e. does it return what `all()` returns by default?